webapps/api/backend/views/vdisks.py

```diff
@@ -232,3 +232,16 @@ def delete(self, vdisk):
         """
         storagerouter = StorageRouter(vdisk.storagerouter_guid)
         return VDiskController.delete.s(diskguid=vdisk.guid).apply_async(routing_key="sr.{0}".format(storagerouter.machine_id))
+
+    @action()
+    @log()
+    @required_roles(['read', 'write'])
+    @return_task()
+    @load(VDisk)
+    def schedule_backend_sync(self, vdisk):
+        """
+        Schedule a backend sync on a vdisk
+        :param vdisk: vdisk to schedule a backend sync to
+        :return: TLogName associated with the data sent off to the backend
+        """
+        return VDiskController.schedule_backend_sync.delay(vdisk_guid=vdisk.guid)
```

Comment:
Just to be sure, but the backend sync should not be triggered on the storagerouter hosting the volume?